### DISCUSSION TITLE
BulkActions and shared tables

### DISCUSSION THREAD

**Discussion by Rattone** [filamentphp/filament#8163] [Help]
Hi! Is it possible to hide or override the bulkActions of a shared table?
Also I am able to add new bulkActions but not within the "BulkActionGroup" (Actions) of the parent table...

(same problem trying to remove the emptyStateActions)

**Comment by pboivin** [comment]
How are you sharing the table? Can you provide some code to show what you've been trying?

**Reply by Rattone** [reply]
I'm talking about [this](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#sharing-a-resources-form-and-table-with-a-relation-manager) and the following paragraphs

For example I've something like this, calling the bulkActions/emptyStateActions/... methods on the resource table allows adding but not removing/hiding/overriding

```
return UserResource::table($table)
    ->columns([
        Tables\Columns\TextColumn::make('pivot.created_at'),
    ])
    ->bulkActions([])
    ->emptyStateActions([]);
```

**Reply by pboivin** [reply]
I'm not sure if this is the best solution, but maybe you could pass parameters to the `table()` function for this specific case?
```php
return UserResource::table($table, withBulkActions: false)
    ->columns([
        Tables\Columns\TextColumn::make('pivot.created_at'),
    ])
    // ...
```
